data/meta/vaults/1/0xc5F3D11580c41cD07104e9AF154Fc6428bb93c73.json

```diff
@@ -11,5 +11,11 @@
   "retired": false,
   "apyTypeOverride": "new",
   "displayName": "Balancer Boosted AAVE USD",
-  "type": "Balancer"
+  "classification": {
+    "isAutomated": true,
+    "isPool": true,
+    "poolProvider": "Balancer",
+    "stability": "Stable",
```

Comment:
correlated as its 3 pools of interest bearing and non-interest bearing assets on bal in a metapool